digits/model/tasks/train.py

```diff
@@ -431,7 +435,7 @@ def infer_many(self, data, model_epoch=None):
         """
         return None
 
-    def get_snapshot(self, epoch=-1):
+    def get_snapshot(self, epoch=-1, download=False):
```

Comment:
the flag was added here even though it's not used in order to keep a consistent interface between Tensorflow and other frameworks.